example-indexer/src/metrics.rs

```diff
@@ -0,0 +1,56 @@
+use actix_web::{get, App, HttpServer, Responder};
+use prometheus::{Encoder, IntGauge, Opts};
+
+use crate::LOGGING_PREFIX;
+
+type Result<T, E> = std::result::Result<T, E>;
+
+
+fn try_create_int_gauge(name: &str, help: &str) -> Result<IntGauge, prometheus::Error> {
+    let opts = Opts::new(name, help);
+    let gauge = IntGauge::with_opts(opts)?;
+    prometheus::register(Box::new(gauge.clone()))?;
+    Ok(gauge)
+}
+
+lazy_static! {
+    pub(crate) static ref LATEST_BLOCK_HEIGHT: IntGauge = try_create_int_gauge(
+        "nearsocial_indexer_latest_block_height",
+        "Last seen block height by indexer"
+    )
+    .unwrap();
+}
+
+#[get("/latest_block_height")]
```

Comment:
The approach according to a convention is to have only 1 endpoint `/metrics` that exposes all the metrics in Prometheus-compatible format. Since, at the moment you have only one metric it should be a big problem to change.

Reply:
Yeah, actually you return it as I've described. So just a simple change
```suggestion
#[get("/metrics")]
```